### DISCUSSION TITLE
Is there any plugin to zoom image?

### DISCUSSION THREAD

**Discussion by alefcarlos** [mkdocs/mkdocs#2873] [Q&A] [answered]
Hi!

Is there any plugin to add zoom image functionality? Large diagrams may be hard to read.

Like this one for docsfy: https://docsify.js.org/#/plugins?id=zoom-image

**Comment by pawamoy** [comment, marked as answer]
Let me check, I recently added such a thing to the best of list.
UPDATE: yup: https://github.com/blueswen/mkdocs-glightbox

**Reply by alefcarlos** [reply]
Hey! Thats awesome ! Thank you!
